coverage/index.html

```diff
@@ -282,319 +305,323 @@ <h2 id="roles_with_more_than_one_label">Roles with More than One Guidance or Exa
+                <li><a href="..\examples\toolbar\toolbar.html">Toolbar</a></li>
             </ul>
 </td>
           </tr>
           <tr>
             <td><code>region</code></td>
-            <td><a href="../aria-practices.html#aria_lh_region">Region</a>
+            <td><ul>
+                <li><a href="../aria-practices.html#aria_lh_region">Region</a></li>
+                <li><a href="../aria-practices.html#aria_lh_region">Region</a></li>
+            </ul>
 </td>
             <td><ul>
-        <li><a href="../examples/accordion/accordion.html">Accordion</a></li>
-        <li><a href="../examples/carousel/carousel-1.html">Auto-Rotating Image Carousel</a></li>
-        <li><a href="../examples/carousel/carousel-2-tablist.html">Auto-Rotating Image Carousel with a Tablist</a></li>
-        <li><a href="../examples/landmarks/region.html">Region Landmark</a></li>
+                <li><a href="..\examples\carousel\carousel-1-prev-next.html">Auto-Rotating Image Carousel  with Buttons for Slide Control</a></li>
+                <li><a href="..\examples\carousel\carousel-2-tablist.html" aria-label="Auto-Rotating Image Carousel with Tabs for Slide Control with High Contrast Support">Auto-Rotating Image Carousel with Tabs for Slide Control</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
+                <li><a href="../examples/landmarks/region.html">Region Landmark</a></li>
             </ul>
 </td>
           </tr>
           <tr>
             <td><code>row</code></td>
             <td><abbr title="none" style="color: gray">-</abbr></td>
             <td><ul>
-        <li><a href="../examples/combobox/grid-combo.html">Editable Combobox with Grid Popup</a></li>
-        <li><a href="../examples/grid/LayoutGrids.html">Layout Grid</a></li>
-        <li><a href="../examples/table/table.html">Table</a></li>
-        <li><a href="../examples/treegrid/treegrid-1.html">Treegrid Email Inbox</a></li>
+                <li><a href="..\examples\combobox\combobox-datepicker.html" aria-label="Date Picker Combobox with High Contrast Support">Date Picker Combobox</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
+                <li><a href="..\examples\dialog-modal\datepicker-dialog.html" aria-label="Date Picker Dialog with High Contrast Support">Date Picker Dialog</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
+                <li><a href="..\examples\grid\dataGrids.html">Data Grid</a></li>
             </ul>
 </td>
           </tr>
           <tr>
-            <td><code>slider</code></td>
+            <td><code>search</code></td>
             <td><ul>
-        <li><a href="../aria-practices.html#slider">Slider</a></li>
-        <li><a href="../aria-practices.html#slidertwothumb">Slider (Multi-Thumb)</a></li>
+                <li><a href="../aria-practices.html#aria_lh_search">Search</a></li>
+                <li><a href="../aria-practices.html#aria_lh_search">Search</a></li>
```

Comment:
There's something slightly odd here and in a couple other places, where it seems like it's generating duplicate links for several examples (here, and in contentinfo/complementary, and maybe others). I can take a closer look at why that's happening later, if you haven't already gotten to it by then 😄

Reply:
@smhigley 
Seems to be related to landmarks, thanks for finding.

Reply:
@smhigley 
Found the problem and fixed the issues, thank you for pointing them out.  Let me know if you see any other issues.